### ISSUE TITLE
App will not open

### ISSUE THREAD

**Issue by teeeeeepot** [Vita3K/Vita3K-Android#336]
Hi, im on Android Pie (9.0) and whenever i try opening vita3k the app crashes after a few seconds. Please help

**Comment by netxblade** [comment]
Yeah I have also have that problem I am on oppo a12 (9.0) and please fix it
